src/style.js

```diff
@@ -0,0 +1,47 @@
+import {Fill, Stroke, Text, Style as OpenLayersStyle, Circle as CircleStyle} from 'ol/style.js';
+
+const _style = Symbol('style')
+
+class ToolStyle {  
+  
+    constructor(styleOptions) {
+
+      const style = new OpenLayersStyle({
+        stroke: new Stroke({
+          color: styleOptions.stroke.color !== undefined ? styleOptions.stroke.color : 'green',
+          width: styleOptions.stroke.width !== undefined ? styleOptions.stroke.width : 1
+        }),
+        fill: new Fill({
+          color: styleOptions.fill.color !== undefined ? styleOptions.fill.color : 'rgba(0, 255, 0, 0.1)'
+        }),
+        image: new CircleStyle({
```

Comment:
What do we need `image` (instance of `CircleStyle`) for?

Reply:
CircleStyle is what is going to render the point tool. Up to now, if you remove the CircleStyle, the point will not be drawn.